src/SayMore/UI/LowLevelControls/Slider.cs

```diff
@@ -170,8 +173,11 @@ public bool SetValueWithoutEvent(float value)
 		{
 			value = (float)Math.Round(value, 4);
 
-			if (value < 0 || value > Maximum)
-				throw new ArgumentOutOfRangeException("value");
+			if (value < 0)
+				value = 0;
```

Comment:
Tom I think you might like to have Sentry wired up to SayMore. LSDEV is running our own instance now. I've found with lameta that it's nice for *me* to get notice of errors that realistically, are meaningless to users and which they wouldn't tell me about.

I'll make an organization in case you want to go this direction.

See https://sentry.keyman.com/saymore/c/getting-started/csharp/